### DISCUSSION TITLE
How to use as CMS?

### DISCUSSION THREAD

**Discussion by mikkelking** [react-page/react-page#980] [Q&A]
This is more of a question than a feature request - I am interested to know how you used react-page as a CMS to edit the pages ? How practical is it to maintain a web site using it?

**Comment by macrozone** [comment]
i move that to discussion

**Comment by macrozone** [comment]
We use it together with [react-admin](https://marmelab.com/react-admin/) and a prisma data store (any data store does the job) with great success. (one example: https://www.veloplus.ch/)

I added an example how that could look like here: https://react-page.github.io/examples/reactadmin

You can add ReactPage to any web-app as alternative to a normal rich-text-editor.

**Comment by albert-dm** [comment]
I'm also interested
I would like to use it with Next.js

**Reply by macrozone** [reply]
you can use it with nextjs (e.g. the examples https://react-page.github.io/ is a static nextjs export).

You can use ReactPage in any "backoffice/admin" page of your nextjs-app as well as in any nextjs page to display the content.  You can use any data-store that you like (I usually used apollo-client and -server for that). ReactPage itself is just like a form-field.

If you need further guidance (that goes beyond what ReactPage solves), I could also offer paid support by e-mail (maw@panter.ch).

I would also like to hear from others how they integrate and use ReactPage in their apps!
